<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>administracion_usuario</class>
 <widget class="QDialog" name="administracion_usuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administracion de usuario</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>349</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="5">
     <widget class="QListWidget" name="list_usuario"/>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="Crear_usuario">
      <property name="text">
       <string>Crear</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="Actualizar_usuario">
      <property name="text">
       <string>Actualizar</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="Consultar_usuario">
      <property name="text">
       <string>Consultar</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Eliminar_usuario">
      <property name="text">
       <string>Eliminar </string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="Cancelar_usuario">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
